iso22133.c

```diff
@@ -145,9 +145,8 @@ typedef struct {
 	uint16_t StartTimeValueIdU16;
 	uint16_t StartTimeContentLengthU16;
 	uint32_t StartTimeU32;
-	uint16_t GPSWeekValueID;
-	uint16_t GPSWeekContentLength;
 	uint16_t GPSWeek;
+	uint16_t TrajID;
```

Comment:
```suggestion
	uint16_t TrajId;
```

To keep it consistent with `StartTimeValueIdU16`

Reply:
TrajID is used in the documentation.